meetings/2024-06-12.md

```diff
@@ -28,11 +35,123 @@
+* JZ: Sarven and VB have similar objections to mine. If you can convince all of us, ok, but you didn't do that yet.
+* eP: maybe vote next week, try to reach consensus. SC and VB do not host webid doc in the solid server.
+* TT: I think it makes some sense do divorce the webid doc from solid storage. Nothing says, that you MUST use webid as the identification methods. Focusing on webid puts roadblocks in our way. It's the only one listed, but it doesn't say it MUST be used.
+* eP: there was some conversation about using DIDs. I think it's opening a big can of worms. All implementations currently use WebID. If want to use something else too, we should provide such examples.
+* TT: DIDs certainly are a possibility. Security concerns should go againt Webid, not against Solid.
+* eP: It's not explicit to WebID, it's related to Solid's reliance on WebID. An attacker could inject something in the webid profile document and then get access to your data.
+* TT: bad connection.
+* TT: there is a circular reference. 
+* HZ: I agree and it should be priority, but we also are deviating from the topic. I also understand Jeff and it is also a priority. I also agree with Pavlik that what Jeff requests shouldn't block that PR. How do we move forward?
+* HZ: Jeff, Is it a strong objection or preference?
+* JZ: I think Sarven and Virginia should be present. Some concerns need to be addressed.
+* HZ: Do you agree that it is orthogonal to your issue?
+* JZ: No I don't agree
+* HZ: From architecture point of view I agree with Pavlik. If you store WebID document on the Solid Pod, if pod is not available the webid document is also not available. But I'm not going to die on that hill.
+* JZ: There are 50k pods on solidcommunity.net those people and those pods I'm concern about. It is not orthogonal. There is an existing situation of knowledgable people having WebIDs out of the Solid server. At minimum Sarven and Virginia need to be present.
+* HZ: I agree with all of you, let's take it offline.
+* Ted: One of the challenges I had in PR9, the structure of the document needs to be more consumable and actionable. There is a blur between weaknesses and how to address them. The countermeasures are at the same level as the weaknesses. One is a theft of the token, there is another with theft of the credentials. There should be weakenss, attack and countermeasure that prevents it. And considerationsa about that weakenss. 
+* eP: It will be easier to structure based on more than one weakness.
+* HZ: I see more thought about infrastructure and not only focus on p2p
```

Comment:
```suggestion
* Ted: One of the challenges I had with PR9 is that the structure of the document needs to be more consumable and actionable. There is a blur between weaknesses and how to address them. The countermeasures section is at the same level as the individual weakness sections. One is a theft of the token; there is another with theft of credentials. There should be weakness, attack, and countermeasure that prevents it. And considerations about that weakness. 
* eP: It will be easier to structure based on more than one weakness.
* HZ: I see more thought about infrastructure and not only focus on p2p.
```